router/aspects/request_timeout_handler.go

```diff
@@ -23,20 +27,25 @@ func QueuedRequestTimeout(f httprouter.Handle, reqTimeoutHeaders config.RequestT
 		reqTimeFloat, reqTimeFloatErr := strconv.ParseFloat(reqTimeInQueue, 64)
 		reqTimeoutFloat, reqTimeoutFloatErr := strconv.ParseFloat(reqTimeout, 64)
 
+		reqTimeDuration := time.Duration(reqTimeFloat * NanosecondsInSecond)
```

Comment:
I assume this means that the header has request time in seconds.

Reply:
Yes, fixed